include/deal.II/lac/trilinos_precondition.h

```diff
@@ -1558,7 +1603,7 @@ namespace TrilinosWrappers
     /**
      * A copy of the deal.II matrix into Trilinos format.
      */
-    std_cxx11::shared_ptr<SparseMatrix> trilinos_matrix;
+    std_cxx11::shared_ptr<SparseMatrix> _trilinos_matrix;
```

Comment:
We don't own the namespace of names that start with an underscore, and we generally don't name members that way. What caused you to rename the member? Same in line 1835.